pkgs/development/python-modules/pyqldb/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, boto3, ion-python, ionhash }:
+
+buildPythonPackage rec {
+  pname = "pyqldb";
+  version = "3.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Xr170hASJ3O+fbkCq4Ay/m42iSueNTVOG1u9CYeSfvI=";
+  };
+
+  propagatedBuildInputs = [ boto3 ion-python ionhash ];
+
+  # No tests in PyPI tarball
```

Comment:
but in GitHub https://github.com/awslabs/amazon-qldb-driver-python/tree/master/tests, please fetch from there and add a pythonImportsCheck.